terraform/files/bin/enable-cilium-sg-kube.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 . ~/bin/cccfg.inc
-SGS=$(openstack security group list -f value -c ID -c Name | grep "k8s-cluster-default-${CLUSTER_NAME}-secgroup-")
+SGS=$(openstack security group list -f value -c ID -c Name | grep "k8s-cluster-\(default-${CLUSTER_NAME}\|${CLUSTER_NAME}-${CLUSTER_NAME}\)-secgroup-")
```

Comment:
Same as above: This is done to support both old (default namespace) and new (`CLUSTER_NAME` namespace) clusters?